### PR TITLE
Jails messages review

### PR DESCRIPTION
**Pull request by Mrt134**

### CODE REVIEW

**Comment by Mrt134** [comment]
rebuild

File: src/app/pages/jails/jail-add/jail-add.component.ts [hunk 1]
```diff
@@ -1148,11 +1148,11 @@ export class JailAddComponent implements OnInit {
               }
             },
             (res_remote) => {
-              this.dialogService.errorReport(T('Error: Get remote release choices failed'), res_remote.reason, res_remote.trace.formatted);
+              this.dialogService.errorReport(T('Error: Displaying remote releases failed.'), res_remote.reason, res_remote.trace.formatted);
```
Comment (wblock) [inline src/app/pages/jails/jail-add/jail-add.component.ts]: "Fetching remote release choices failed."

File: src/app/pages/jails/jail-list/jail-list.component.ts [hunk 2]
```diff
@@ -132,17 +132,20 @@ export class JailListComponent implements OnInit {
   constructor(protected router: Router, protected rest: RestService, protected ws: WebSocketService, 
     protected loader: AppLoaderService, protected dialogService: DialogService, private translate: TranslateService) {}
 
-  public tooltipMsg: any = T("Choose an existing ZFS Pool to allow the iocage jail manager \
-  to create a /iocage dataset in the selected pool. The '/iocage' dataset may not be visible \
-  until after the first jail is created. iocage uses this dataset to store FreeBSD RELEASES \
-  and all other jail data. To create a new ZFS Pool, navigate Storage/Volumes and click 'Create ZFS Pool'.");
+  public tooltipMsg: any = T("Choose an existing ZFS Pool to allow the \
```
Comment (wblock) [inline src/app/pages/jails/jail-list/jail-list.component.ts]: "Choose a pool where the iocage jail manager can create the /iocage dataset. The /iocage..."

File: src/app/pages/jails/jail-list/jail-list.component.ts [hunk 3]
```diff
@@ -132,17 +132,20 @@ export class JailListComponent implements OnInit {
...
+                              iocage jail manager to create the /iocage \
+                              dataset in the selected pool. The /iocage \
+                              dataset might not be visible until after \
+                              the first jail is created. iocage uses \
+                              this dataset to store FreeBSD RELEASES \
```
Comment (wblock) [inline src/app/pages/jails/jail-list/jail-list.component.ts]: s/RELEASES/releases/

File: src/app/pages/jails/jail-wizard/jail-wizard.component.ts [hunk 4]
```diff
@@ -255,11 +255,11 @@ export class JailWizardComponent {
                 }
               },
               (res_remote) => {
-                this.dialogService.errorReport(T('Error: Get remote release choices failed'), res_remote.reason, res_remote.trace.formatted);
+                this.dialogService.errorReport(T('Error: Display remote releases failed.'), res_remote.reason, res_remote.trace.formatted);
```
Comment (wblock) [inline src/app/pages/jails/jail-wizard/jail-wizard.component.ts]: 'Fetching remote release choices failed.'

**Comment by Mrt134** [comment]
rebuild

**Comment by Mrt134** [comment]
rebuild

**Pull request merged**